From e2cc40475092e7af4bb92b294b28046a33abf76b Mon Sep 17 00:00:00 2001
From: Stella Laurenzo <stellaraccident@gmail.com>
Date: Fri, 30 May 2025 18:19:06 -0700
Subject: [PATCH 2/2] [next-cmake] Link against roctx64.

Prior, we were finding it but left it out of the link.
---
 next-cmake/CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/next-cmake/CMakeLists.txt b/next-cmake/CMakeLists.txt
index d2ba9d75..3ba8eabc 100644
--- a/next-cmake/CMakeLists.txt
+++ b/next-cmake/CMakeLists.txt
@@ -121,6 +121,8 @@ if(HIPBLASLT_ENABLE_HOST)
     endif()
     if(HIPBLASLT_ENABLE_MARKER)
         find_library(rocTracer roctx64 REQUIRED)
+    else()
+        set(rocTracer)
     endif()
 endif()
 if(HIPBLASLT_REQUIRE_ROCM_SMI)
@@ -210,6 +212,7 @@ if(HIPBLASLT_ENABLE_HOST)
             hip::device
             rocisa::rocisa-cpp
             ${CMAKE_DL_LIBS}
+            ${rocTracer}
     )
 
     if(HIPBLASLT_ENABLE_ASAN)
-- 
2.43.0

